src/context/scriptProviderContext.ts

```diff
@@ -35,8 +35,7 @@ export function destroySDKScript(reactPayPalScriptID: string): void {
         `script[${SCRIPT_ID}="${reactPayPalScriptID}"]`
     );
 
-    if (scriptNode != null && scriptNode.parentNode)
-        scriptNode.parentNode.removeChild(scriptNode);
+    if (scriptNode?.parentNode) scriptNode.parentNode.removeChild(scriptNode);
```

Comment:
```suggestion
    if (scriptNode?.parentNode) {
        scriptNode.parentNode.removeChild(scriptNode);
    }
```

very minor nitpick so feel free to reject this suggestion. just throwing it out there as i prefer all conditionals in the bracketed syntax for easier readability. the minifier can simplifier it down later

Reply:
Not a problem at all. I'll change to keep the bracketed syntax, I guess is the standard.